libs/payments/customer/src/lib/invoice.factories.ts

```diff
@@ -21,14 +20,8 @@ export const InvoicePreviewFactory = (
     '-' +
     faker.string.numeric({ length: 4, allowLeadingZeros: true }),
   nextInvoiceDate: faker.date.future().getDate(),
-  ...override,
-});
-
-export const InvoicePreviewForUpgradeFactory = (
-  override?: Partial<InvoicePreviewForUpgrade>
-): InvoicePreviewForUpgrade => ({
-  ...InvoicePreviewFactory(override),
-  oneTimeCharge: faker.number.int({ min: 1, max: 1000 }),
-  oneTimeChargeSubtotal: faker.number.int({ min: 1, max: 1000 }),
+  amountDue: faker.number.int({ min: 0, max: 1000 }),
```

Comment:
Please update this to have `min: 1`, rather than `min: 0`, as an unexpected value of 0 here can have unintended consequences for how we process transactions